### PR TITLE
Link to Discourse first in "Help with Documentation" section (take 2)

### PR DESCRIPTION
**Pull request by Mariatta**
The Docs Discourse forum is the preferred way.
I reordered the sentence so that the discourse forum is mentioned first.

----
:books: Documentation preview :books:: https://cpython-devguide--1118.org.readthedocs.build/

### CODE REVIEW

**Review by erlend-aasland — APPROVED** [review]

File: documentation/help-documenting.rst [hunk 1]
```diff
@@ -36,11 +36,11 @@ and :ref:`maintenance <maintbranch>` branches at https://docs.python.org/dev/.
...
-to the `docs@python.org <https://mail.python.org/mailman3/lists/docs.python.org/>`_
-mailing list and the `Documentation category on the Python Discourse
-<https://discuss.python.org/c/documentation/26>`_,
-where user issues are raised and documentation toolchain, projects, and standards
-are discussed.
+to the `Documentation category on the Python Discourse
+<https://discuss.python.org/c/documentation/26>`_ and the
+`docs@python.org <https://mail.python.org/mailman3/lists/docs.python.org/>`_
+mailing list where user issues are raised and documentation toolchain,
+projects, and standards are discussed.
```
Comment (CAM-Gerlach) [inline documentation/help-documenting.rst:43]:
```suggestion
`docs@python.org <https://mail.python.org/mailman3/lists/docs.python.org/>`_ mailing list
where user issues are raised and documentation toolchain, projects, and standards
are discussed.
```

Remaking my suggestion again to minimize unnecessary diff noise, as was generally agreed in #1018 — this avoids changing the latter two otherwise-untouched lines solely due to hard wrapping.

**Review by CAM-Gerlach — APPROVED** [review]
LGTM with the text, though I did re-make my suggestion to minimize unessesary diff noise. Thanks!
